src/components/dashjs-player/dashjs-player.scss

```diff
@@ -1,4 +1,4 @@
-@import './controlbar.scss';
+@import '../contrib/controlbar.scss';
```

Comment:
it was ok as it is, please remove the files in contrib

Reply:
done